packages/ffe-context-message/less/ffe-context-message.less

```diff
@@ -57,19 +57,14 @@
             @media (prefers-color-scheme: dark) {
                 color: @ffe-farge-svart;
             }
-            a {
```

Comment:
hvis man bruke en ffe-link https://design.sparebank1.no/styleguidist/index.html#!/LinkText . Hvordan vill det se ut da?
Blir det blått på blått? Kunde man bytt ut a mot .ffe-link-text?

Reply:
Eller blir den så mørkeblå att det er innanfør sån kontrastmessig?

Reply:
Blå på blå er for så vidt sånn det vil vises by default i dag, siden denne overstyringen kun gjaldt dark mode i utgangspunktet. Men det er nok ikke helt gjennomtenkt. Er usikker på hvor utbredt det er å bruke tekstlinker i kontekstmeldinger.

Jeg har oppdatert og satt `.ffe-context-message-content .ffe-link-text` til `@ffe-farge-fjell` for nå, så har man i alle fall en mulighet til å bruke lenker. Den bør være mørk nok til å dekke kontrasten på alle kontekstmeldinger (grønn, blå, rød, sand) i både dark og light mode, men vi kan sikkert med fordel se over dette ordentlig ved anledning. :)

Reply:
Når man talar om trollen så brakk akkurat en e2e test av annen grund som viser dette. 
![image](https://user-images.githubusercontent.com/2248579/128204905-9f4fa9e1-a995-4598-8b26-ccbb8347ad4a.png)

Reply:
> Når man talar om trollen så brakk akkurat en e2e test av annen grund som viser dette.

Dette screenshotet viser en melding fra `ffe-message-box` og ikke `ffe-context-message`, så det er neppe relevant for akkurat denne pr-en. :) Det har heller ikke skjedd noen endringer i `ffe-message-box` nylig, men oppdaterte farger er på vei i #1152.

Reply:
Neida, ok jag er osikker på om vi bruker linkar i akkurat dom.